tensorvision/utils.py

```diff
@@ -56,14 +56,18 @@ def set_dirs(hypes, hypes_fname):
         else:
             runs_dir = os.path.join(base_path, 'RUNS')
 
+        # test for project dir
+        if hasattr(FLAGS, 'project') and FLAGS.project is not None:
+            runs_dir = os.path.join(runs_dir, FLAGS.project)
+
```

Comment:
Do you think adding an `else` branch would be a good idea? This could be set to

```
    runs_dir = os.path.join(runs_dir, 'default_project')
```

or something similar. Just in case we rely on having this at some point.